### ISSUE TITLE
Unable to edit pages, posts, or even click on "appearance" on sidebar

### ISSUE THREAD

**Issue by vbrunetti** [ThemeFuse/Unyson#4197]
- PHP 7.4 (though I've tried and get the same results with 7.3 and 5.6. PHP 8.0 crashes everything which is a separate issue)
- Unyson Version 2.7.24
- Wordpress version 5.8.2 (though I've tried to rollback to the last Unyson-supported version of WP 5.5.7 and get the same result)
- Try to edit a page or post and I get the following error:

> This site can’t be reachedThe webpage at "[URL]/wp-admin/post.php?post=3320&action=edit" might be temporarily down or it may have moved permanently to a new web address.
> ERR_HTTP2_PROTOCOL_ERROR

- I've tried changing the permalink settings but that doesn't work
- I've disabled all other plugins but that doesn't work
+ I CAN successfully edit a post or page if I disable Unyson (which makes the live site look all kinds of garbage), publish my edit, and then turn the Unyson plugin back on. Very suboptimal workflow and indicative of an issue localized to Unyson.

Anyone with a similar issue?

**Comment by yura-x** [comment]
Can you please try to check you PHP error log? What errors do you have?

**Comment by vbrunetti** [comment]
Thanks for the reply! I get this from the Debug console in WP:
`NOTICE: wp-content/themes/the-core-parent/framework-customizations/extensions/shortcodes/shortcodes/image-box/views/view.php:129 - Trying to access array offset on value of type bool
require('wp-blog-header.php'), require_once('wp-includes/template-loader.php'), include('/themes/the-core-parent/visual-builder-template.php'), the_content, apply_filters('the_content'), WP_Hook->apply_filters, do_shortcode, preg_replace_callback, do_shortcode_tag, FW_Shortcode->render, FW_Shortcode->_render, fw_render_view, require('/themes/the-core-parent/framework-customizations/extensions/shortcodes/shortcodes/section/views/view.php'), do_shortcode, preg_replace_callback, do_shortcode_tag, FW_Shortcode->render, FW_Shortcode->_render, fw_render_view, require('/plugins/unyson/framework/extensions/shortcodes/shortcodes/row/views/view.php'), do_shortcode, preg_replace_callback, do_shortcode_tag, FW_Shortcode->render, FW_Shortcode->_render, fw_render_view, require('/themes/the-core-parent/framework-customizations/extensions/shortcodes/shortcodes/column/views/view.php'), do_shortcode, preg_replace_callback, do_shortcode_tag, FW_Shortcode->render, FW_Shortcode->_render, fw_render_view, require('/themes/the-core-parent/framework-customizations/extensions/shortcodes/shortcodes/image-box/views/view.php')`

**Comment by vbrunetti** [comment]
...and several more from the log file:

[29-Dec-2021 18:11:09 UTC] PHP Strict Standards:  Declaration of FW_Auto_Install_Upgrader_Skin::feedback() should be compatible with WP_Upgrader_Skin::feedback($string, ...$args) in /home/[USERNAME REDACTED]/public_html/wp-content/themes/the-core-parent/theme-includes/includes/sub-includes/auto-setup/class-fw-auto-install.php on line 1880

[29-Dec-2021 18:11:55 UTC] PHP Warning:  Declaration of FW_Auto_Install_Upgrader_Skin::feedback($string) should be compatible with WP_Upgrader_Skin::feedback($string, ...$args) in /home/[USERNAME REDACTED]/public_html/wp-content/themes/the-core-parent/theme-includes/includes/sub-includes/auto-setup/class-fw-auto-install.php on line 1882

[29-Dec-2021 18:13:11 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/[USERNAME REDACTED]/public_html/wp-content/themes/the-core-parent/framework-customizations/extensions/shortcodes/shortcodes/image-box/views/view.php on line 129

[29-Dec-2021 18:21:10 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/[USERNAME REDACTED]/public_html/wp-content/themes/the-core-parent/theme-includes/helpers.php on line 1413

**Comment by yura-x** [comment]
There are no error level PHP errors.
Try to install 'Classic Editor' plugin.
Try to temporary change your theme and check your post/page edit screen.

**Comment by ipiyushverma** [comment]
> * PHP 7.4 (though I've tried and get the same results with 7.3 and 5.6. PHP 8.0 crashes everything which is a separate issue)
> * Unyson Version 2.7.24
> * Wordpress version 5.8.2 (though I've tried to rollback to the last Unyson-supported version of WP 5.5.7 and get the same result)
> * Try to edit a page or post and I get the following error:
> 
> > This site can’t be reachedThe webpage at "[URL]/wp-admin/post.php?post=3320&action=edit" might be temporarily down or it may have moved permanently to a new web address.
> > ERR_HTTP2_PROTOCOL_ERROR
> 
> * I've tried changing the permalink settings but that doesn't work
> * I've disabled all other plugins but that doesn't work
> 
> * I CAN successfully edit a post or page if I disable Unyson (which makes the live site look all kinds of garbage), publish my edit, and then turn the Unyson plugin back on. Very suboptimal workflow and indicative of an issue localized to Unyson.
> 
> Anyone with a similar issue?

did you get any solution

**Comment by vbrunetti** [comment]
No, and as it turns out, I **CAN'T** successfully edit a page if I disable Unyson, edit the HTML by hand, and then reactivate Unyson. What happens is that I can see the edits live while Unyson is off (but, again, the site looks terrible), and when I switch Unyson back on, the edits disappear, and the content reverts back to original. How can the content be there AND not there at the same time?
So as of now, I'm dead in the water with a completely un-editable website.
